mock banking page/widget/carousel.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+import CustomCard from "./carSlides";
+
+/**
+ * A react component Carousel that renders and contains each individual CustomCard from carSlides.
+ *
+ * The user can interact with it through the arrows or by slidling their cursor to obtain the next card.
+ *
+ * It takes as props the values for each card
+ */
+class Carousels extends React.Component {
+    constructor(props){
+        super()
+        this.props = props;
+    }
+    renderItems() {
```

Comment:
can you add documentation to the methods?

Reply:
Done